contracts/Vault.vy

```diff
@@ -222,6 +222,8 @@ MAX_BPS: constant(uint256) = 10_000  # 100%, or 10k basis points
 #       So the average year has 365 + 97/400 = 365.2425 days
 #       ERROR(Julian): -0.0078
 #       ERROR(Gregorian): -0.0003
+#       A day = 24 * 60 * 60 sec = 86400 sec
+#       365.2425 * 86400 = 31556952.0
```

Comment:
Would be more interesting to know why a year is 365.2425 than a day = 86400 sec

Reply:
yes. old note:
Don't know the relationship between 365.2425 and 31556952